nexus/src/external_api/http_entrypoints.rs

```diff
@@ -7088,7 +7088,10 @@ impl NexusExternalApi for NexusExternalApiImpl {
                 crate::context::op_context_for_external_api(&rqctx).await?;
 
             let head = false;
-            let range = rqctx.range();
```

Comment:
As you can see here, we actually were parsing the range request headers before - just not visibly through the API spec.